lib/init.js

```diff
@@ -0,0 +1,71 @@
+require('../config/env');
+
+const fs = require('fs');
+const nconf = require('nconf');
+const path = require('path');
+
+const defaultRulesDir = nconf.get('STYLEGUIDES_DIR');
+
+function createNewRuleStructure(name, rulesDir = defaultRulesDir) {
+  if (!name) {
+    throw new Error('Rule name must be provided');
+  }
+
+  const dir = path.join(path.resolve(__dirname, `../${rulesDir}`), name);
+
+  if (fs.existsSync(dir)) {
+    throw new Error(`Rule \`${name}\` already exist`);
+  }
+
+  const badExampleBaseDir = path.join(dir, 'bad');
+  const goodExampleBaseDir = path.join(dir, 'good');
+  const badExampleDir = path.join(badExampleBaseDir, 'example');
+  const goodExampleDir = path.join(goodExampleBaseDir, 'example');
+
+  const goodExample = `title: Title of good example
+description: Description of good example
+target: target this example will be tested against
+`;
+
+  const badExample = `title: Title of bad example
+description: Description of bad example
+error: expected error
+target: target this example will be tested against
+`;
+
+  const rule = `title: title of this rule
+intent: intent/description of this rule
+allowedTargets: list of allowed targets
```

Comment:
> list of allowed targets

if this is .yaml, this part could be a comment
Same for functionName

Reply:
changed